### PR TITLE
Create migration guidance for `easing()` function

### LINKED ISSUE #5030: Create migration guidance for `easing()` function

### PR DESCRIPTION
**Pull request by lgriffee**
### WHY are these changes introduced?

Fixes #5030  

### WHAT is this pull request doing?

- Creating a new _Sass functions and mixins_ section in our migration guide
- Adding documentation for the [removal of the `easing()` function](https://github.com/Shopify/polaris-react/pull/4698/files) to our migration guide

### CODE REVIEW

**Review by aaronccasanova — APPROVED** [review]
🎉

File: documentation/guides/migrating-from-v8-to-v9.md [hunk 1]
```diff
@@ -75,6 +75,31 @@ CSS custom properties that have been deprecated can be replaced with the CSS pro
...
+
+### Adding the functions and mixins to your repo
+
+To help you quickly add these functions and mixins back to your repo, we've created a css file with all the removed functions and mixins.
+
+[✨ Amazing mega file linked here ✨]
+
+### Replacing function and mixin instances with values or tokens
+
+To help you replace function and mixin instances in your repo, we've provided the following list of functions/mixins and their value equivalents or new token values.
```
Comment (aveline) [inline documentation/guides/migrating-from-v8-to-v9.md]:
```suggestion
A list of functions/mixins and their value equivalents or new token values.
```

File: documentation/guides/migrating-from-v8-to-v9.md [hunk 2]
```diff
@@ -75,6 +75,31 @@ CSS custom properties that have been deprecated can be replaced with the CSS pro
...
+
+#### `easing()`
+
+| Function                 | Replacement Value/Token              |
+| ------------------------ | ------------------------------------ |
+| easing()<br>easing(base) | var(--p-ease)                        |
```
Comment (aveline) [inline documentation/guides/migrating-from-v8-to-v9.md]:
```suggestion
| `easing()`<br>`easing(base)` | `var(--p-ease)`                        |
```

Comment (aveline) [reply documentation/guides/migrating-from-v8-to-v9.md]: This is just a suggestion, not sure if we want to do this for all of them.

**Review by aveline — APPROVED** [review]

File: documentation/guides/migrating-from-v8-to-v9.md (hunk 2 above)
Comment (lgriffee) [reply documentation/guides/migrating-from-v8-to-v9.md]: Good call! Yeah I think it makes sense to do it for all of them 👍🏻

**Pull request merged**